modules/nf-core/rundbcan/cazymeannotation/main.nf

```diff
@@ -0,0 +1,65 @@
+process RUNDBCAN_CAZYMEANNOTATION {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
```

Comment:
```suggestion
    // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
    conda "${moduleDir}/environment.yml"
```

Reply:
Assuming that there is indeed no way to get the version information from the tool?
If it is your tool as I think it is, could you add a --version command to it in your next release?